libs/sdk-flutter/lib/breez_sdk.dart

```diff
@@ -24,6 +24,9 @@ class BreezSDK {
   void initialize() {
     /// Listen to BreezEvent's(new block, invoice paid, synced)
     _lnToolkit.breezEventsStream().listen((event) async {
+      if (event is BreezEvent_NewBlock) {
+        await fetchNodeData();
```

Comment:
As far as I see the sync in the sdk is always called on new block so I am not sure we need that.
We already listen to sync and fetching the node data.

Reply:
True, but when testing this the latest node date isn't received by the flutter side in c-breez even though if it's present in the sdk.

Reply:
We need to understand why.

Reply:
Disregard what I wrote, the new state is emitted when a new block arrives.